getting_started.md

```diff
@@ -0,0 +1,109 @@
+# Getting Started
+
+## Workspace set up and installation
+To get started you will have to make a few decisions and install the repository along with its dependencies. Specifically:
+1. Decide if you would like to develop your submission in either Pytorch or Jax.
+2. Set up your workstation or VM. We recommend to use a setup similar to the competition scoring hardware which consists out of 8 V100 GPUs, 350 GB in RAM and 2 TB in storag for dataset storage. For recommendations on setting up your own Cloud VM see [here](https://github.com/mlcommons/algorithmic-efficiency/blob/main/docker/README.md#gcp-integration).
```

Comment:
how much ram do we actually use in the GCP VMs? can you link to https://github.com/mlcommons/algorithmic-efficiency/blob/main/RULES.md#benchmarking-hardware (and edit the link there to point to https://cloud.google.com/compute/docs/gpus#nvidia_v100_gpus and also include the actual amount of ram we use?)

the AWS instance says it has 488GB of host RAM.

Reply:
The GCP instances I used for timing have 240 GB RAM